shared/metrics/src/main/java/io/pravega/shared/metrics/MetricProxy.java

```diff
@@ -79,9 +79,6 @@ void updateInstance(T instance) {
     }
 
     protected T getInstance() {
-        if (!closed.get()) {
```

Comment:
I think it would have sufficed to simply fix the condition here. But I agree that probably nobody will look for these warnings in the logs anyway so we might as well get rid of it completely.